src/components/dashboard/SendQRSummary.js

```diff
@@ -46,6 +46,7 @@ const SendQRSummary = (props: AmountProps) => {
   }
   const sendGD = async () => {
     try {
+      setLoading(true)
       const receipt = await goodWallet.sendAmount(to, amount, {
         onTransactionHash: hash => {
```

Comment:
Doing this will prevent having the event in the feed until is confirmed. Is this correct? shouldn't we keep this in the feed and update status when we get updates? Example: transaction pending?

Reply:
yes I thought about it also, we'll refactor the behaviour for all TXs to show pending and notify user once completed.
currently the flow is that for all TXs the button shows loading until tx completes